### DISCUSSION TITLE
Issue template for *unusable* websites?

### DISCUSSION THREAD

**Discussion by NicoWeio** [OhMyGuus/I-Still-Dont-Care-About-Cookies#6414] [Ideas]
I just opened https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/6413 because a client of mine could not access a website anymore and got HTTP 500 pages instead. Only through some investigation, we found this extension to be the cause. Wouldn't such scenarios justify 

- a separate issue template
- or a clarification in an existing issue template
- plus a separate issue label

so that downright unusable websites get some extra attention?

**Comment by inson1** [comment]
agree + button in extension for that
